src/components/button/button.scss

```diff
@@ -1,91 +1,229 @@
+/**
+ * Copyright Kyndryl, Inc. 2023
+ */
+
+@import '../../scss/variables/index';
 @import '../../scss/global.scss';
 
-:host {
-  display: inline-block;
+// Base Button Styles
+.kd-btn {
+  -webkit-transition: border-color 0.3s, color 0.3s, background-color 0.3s;
+  transition: border-color 0.3s, color 0.3s, background-color 0.3s;
+  font-size: var(--kd-font-size-body-2-md);
+  font-weight: var(--kd-font-weight-regular);
+  font-family: var(--kd-font-family-secondary);
+  line-height: var(--kd-line-height-body-2-md);
```

Comment:
Use one of the typography mixins here instead.

Reply:
I tried to look for a typography mixin that meet all the requirements but wasn't able to.
Besides that mixin includes all these vars, these values should be fixed/persist over screen sizes (confirmed with UX). So if you can help suggest one, it'd be great.

Reply:
Thanks